### ISSUE TITLE
Question about the result in DreamVideo

### ISSUE THREAD

**Issue by RongPiKing** [damo-vilab/i2vgen-xl#123]
Why are my results after training inference so poor?
The example I use is “a * is playing guitar”
https://github.com/ali-vilab/VGen/assets/67503039/83b3358e-3385-40d6-b6c4-7d1739bf5ae8

**Comment by weilllllls** [comment]
The results appear to show that the subject ID is overly preserved. You might try reducing the `identity_adapter_index` and increasing the `motion_adapter_index` during inference.

**Comment by Amnesiae** [comment]
Thank you for your answer！I did not modify any code, so do I need to adjust the parameters in the code myself to achieve the effect in the paper?

**Comment by weilllllls** [comment]
Yes, you will need to adjust the hyperparameters to suit your specific cases and we also provide some examples.
